Eshop.Client/src/App.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react'
+
+type CategoryType = {
+  id: number
+  name: string
+}
+
+type ProductType = {
+  id: number
+  title: string
+  description: string
+  price: number
+}
+
+
+
+function App() {
+  const [categories, setCategories] = useState<CategoryType[]>([])
+  useEffect(()=>{
+    const loadCategories = async() =>{
+      const categoryResult = await (await fetch('https://localhost:7203/Category')).json()
+
+      setCategories(categoryResult)
+    }
+
+    loadCategories();
+  }, [])
+
+  const [products, setProducts] = useState<ProductType[]>([])
+  useEffect(()=>{
+    const loadProducts = async() =>{
+      const productResult = await (await fetch('https://localhost:7203/Category')).json()
```

Comment:
![image](https://github.com/akonaweb/Eshop/assets/162015065/22174bec-1604-4184-9670-b107d59fb0fe)

You have wrong endpoint here

Reply:
I fixed it  BUT 
I dont have installed React properly on my second machine so I couldnt test it and I cant install it because of very slow internet -- So Ignore it if there is an ocean of errors until I check it home est. time 7pm of this day

Reply:
alright works on my main pc, so fixed - but I imported react from react because I got errors at the result
div and li tags